package.json

```diff
@@ -218,17 +218,17 @@
     "tsickle": "0.39.1",
     "tslint": "^6.1.3",
     "tsutils": "^3.21.0",
```

Comment:
Is tsutils still used somewhere now?

Reply:
The only thing we're using it for now is to iterate over the comments of a node which we may be able to refactor as well, but the `tsutils` implementation seemed a bit more involved to me so I decided to leave it.